transcripts/how-to-apply-for-a-federal-job-with-the-cfpb/index.html

```diff
@@ -0,0 +1,102 @@
+{% extends "layout-full.html" %}
+
+{% block title -%}
+    How to Apply For a Federal Job with the CFPB
+{%- endblock %}
+
+{% block desc -%}
+    Transcript for the video "How to Apply For a Federal Job with the CFPB."
+{%- endblock %}
+
+{% block content_modifiers -%}
+    {{ super() }}
+    transcripts
```

Comment:
Removed in 62dd451.

Right now the lines are too long. We had a readability guideline proposed in the design manual here: https://github.com/cfpb/design-manual/issues/275#issuecomment-119292792

I posted a comment here about it. If we can't make the style, I'm changing it back to "transcripts" and add a rule there for limit line-length.